src/cmdline.rs

```diff
@@ -215,21 +233,80 @@ pub struct InstallConfig {
     #[structopt(long)]
     pub insecure_ignition: bool,
     /// Base URL for Fedora CoreOS stream metadata
+    #[serde_as(as = "Option<DisplayFromStr>")]
     #[structopt(long, value_name = "URL")]
     pub stream_base_url: Option<Url>,
     /// Don't clear partition table on error
     #[structopt(long)]
     pub preserve_on_error: bool,
     /// Fetch retries, or "infinite"
+    #[serde(skip_serializing_if = "is_default")]
     #[structopt(long, value_name = "N", default_value)]
     pub fetch_retries: FetchRetries,
 
     // positional args
+    // use #[serde(rename(serialize = "_[...]"))] on each of these
```

Comment:
Can you expand that comment with the why? Basically something like:

```suggestion
    // use #[serde(rename(serialize = "_[...]"))] on each of these so that
    // we don't serialize it as e.g. `--device DEV`
```